.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = tab
+indent_size = 4
```

Comment:
The `package.json` file usually is indented with `2`, instead of `4`.

It's fine to keep the `4` for all files, but add an exception for the `package.json`. Otherwise, every time you change something manually - the indentation of the whole file will change.